components/wb-fieldflow/fieldflow-doc-en.html

```diff
@@ -331,14 +331,31 @@ <h3>Set on the plugin <code>&lt;div class=&quot;wb-fieldflow&quot; data-wb-field
 			</tr>
 			<tr>
 				<td>inline</td>
-				<td>Indicate to render an inline interface. Available only for <code>{&quot;renderas&quot;:&quot;checkbox&quot;}</code> and <code>{&quot;renderas&quot;:&quot;radio&quot;}</code> </td>
-				<td><code>data-wb-fieldflow='{&quot;renderas&quot;: &quot;<em>[radio|checkbox]</em>&quot;, &quot;inline&quot;: <em>true</em>}</code></td>
+				<td>Indicate to render an inline interface. Available only for <code>{&quot;renderas&quot;:&quot;checkbox&quot;}</code> and <code>{&quot;renderas&quot;:&quot;radio&quot;}</code> . <br><br> Can also be applied to render an inline submit button <code>{&quot;inline&quot;:&quot;true&quot;}</code></td>
+				<td><code>data-wb-fieldflow='{&quot;renderas&quot;: &quot;<em>[radio|checkbox]</em>&quot;, &quot;inline&quot;: <em>true</em>}</code><br><br>
+					<code>data-wb-fieldflow='{ &quot;inline&quot;: <em>true</em>}</code>
+				</td>
 				<td>
 					<dl>
 						<dt>false (default):</dt>
 						<dd>Render one checkbox or radio per line</dd>
 						<dt>true</dt>
-						<dd>Checkbox or radio are side by side.</dd>
+						<dd>Checkbox or radio are side by side</dd>
+						<dd>(if applied without renderas) Submit button is inline.</dd>
+					</dl>
+				</td>
+			</tr>
+			<tr>
+				<td>btnColor</td>
+				<td>Color for inline button. Available only for inline buttons <code>{&quot;btnColor&quot;:&quot;primary&quot;}</code>.<br> Values examples for "btnColor" option:"default", "primary", "success", "info", "warning", "danger" </td>
+				<td><code>data-wb-fieldflow='{&quot;btnColor&quot;: <em>&quot;[ button class]&quot;</em>}</code>
+				</td>
+				<td>
+					<dl>						
+						<dt>Default:</dt>
+						<dd>default class is "btn-default"</dd>	
+						<dt>String:</dt>
+						<dd>Any valid class attribute as ex. :"btn-success"</dd>			
```

Comment:
Please list the classes from Bootstrap 3, without the "btn-" prefix, such as: success, warning, danger, etc.

Reply:
done. the following list is now added : "default","primary","success","info","warning","danger","link"